textxjinja/__init__.py

```diff
@@ -136,13 +148,24 @@ def generate_file(src_rel_file, src_file, target_file):
                 except SkipFile:
                     continue
 
-                # Create necessary folders.
-                try:
-                    os.makedirs(os.path.dirname(target_file))
-                except FileExistsError:
-                    pass
-
-                generate_file(src_rel_file, src_file, target_file)
+                # Check if target_file is dictionary
```

Comment:
This comment states the obvious so it's not useful. Either remove it or replace by stating why we do this check.

Reply:
I've changed the comment. Is it more meaningful now or should I remove it?